### DISCUSSION TITLE
Feature Request: `getText` method in `@tiptap/html` package

### DISCUSSION THREAD

**Discussion by tibineagu** [ueberdosis/tiptap#4518] [Feature Requests]
### Description

The [`@tiptap/html` package](https://github.com/ueberdosis/tiptap/tree/main/packages/html/src) has some very useful utilities for generating HTML out of the prosemirror JSON, or the JSON out of HTML.

However, we found that there could be a big use-case for extracting the plaintext version of an HTML/JSON on the server as well.

This is currently only possible on the client, with [`editor.getText()`](https://tiptap.dev/api/editor#get-text).

### Use Case

Several use-cases include:
- generating the meta description for the page
- getting an excerpt/truncated version of the original document to use with a "collapsed" state of an article
- sending the document over to an LLM for obtaining a summary of the content

### Type

New feature

**Comment by tibineagu** [comment]
Hmmm i think i jumped the gun with this, i believe [`generateText` from `@tiptap/core`](https://github.com/ueberdosis/tiptap/blob/main/packages/core/src/helpers/generateText.ts) (added in #1875 to close #1428) does exactly what I need and works server-side as well.

**Reply by nperez0111** [reply]
Yep that would be the way to do it, with the `generateText` method
